build/Build.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Resolves CI build failure mentioned in: https://github.com/qdrant/qdrant-dotnet/pull/77#discussion_r1916378669